Manito/Manito/Screens/Detail-Wait/DetailEditViewController.swift

```diff
@@ -99,63 +110,125 @@ final class DetailEditViewController: UIViewController {
+        output.overMember
+            .receive(on: DispatchQueue.main)
+            .filter { !$0 }
+            .sink(receiveValue: { [weak self] value in
+                self?.showAlertOverMember(value)
+            })
+            .store(in: &self.cancellable)
+        
+        output.changeSuccess
+            .receive(on: DispatchQueue.main)
+            .sink(receiveCompletion: { [weak self] result in
+                switch result {
+                case .finished: return
+                case .failure(let error):
+                    self?.showErrorAlert(error.localizedDescription)
+                }
+            }, receiveValue: { [weak self] statusCode in
+                switch statusCode {
                 case 200..<300:
-                    completionHandler(.success(()))
+                    self?.detailWaitDelegate?.didTappedChangeButton()
+                    self?.dismiss(animated: true)
                 default:
-                    completionHandler(.failure((.unknownError)))
+                    return
                 }
-            } catch NetworkError.serverError {
-                completionHandler(.failure(.serverError))
-            } catch NetworkError.clientError(let message) {
-                completionHandler(.failure(.clientError(message: message)))
-            }
-        }
+            })
+            .store(in: &self.cancellable)
+    }
+    
+    private func bindUI() {
+        self.bindCalendar()
+        self.bindSlider()
+        self.bindCancleButton()
+    }
+    
+    private func bindCalendar() {
+        let calendarView = self.detailEditView.calendarView
+        let calendarPublisher = calendarView.startDateTapPublisher.zip(calendarView.endDateTapPublisher)
+        calendarPublisher
+            .sink(receiveValue: { [weak self] startDate, endDate in
+                guard let room = self?.roomPublisher.value else { return }
+                let dto = CreatedRoomInfoRequestDTO(title: room.title,
+                                                    capacity: room.capacity,
+                                                    startDate: startDate,
+                                                    endDate: endDate)
+                self?.roomPublisher.send(dto)
+            })
+            .store(in: &self.cancellable)
+    }
+    
+    private func bindSlider() {
+        self.detailEditView.sliderPublisher
+            .sink(receiveValue: { [weak self] value in
+                guard let room = self?.roomPublisher.value else { return }
+                let dto = CreatedRoomInfoRequestDTO(title: room.title,
+                                                    capacity: Int(value),
+                                                    startDate: room.startDate,
+                                                    endDate: room.endDate)
+                self?.roomPublisher.send(dto)
+            })
+            .store(in: &self.cancellable)
+    }
+    
+    private func bindCancleButton() {
```

Comment:
p1;

```suggestion
    private func bindCancelButton() {
```

변경부탁드립니다!

Reply:
ㅋㅋㅋㅋㅋㅋ 민망하군요...
[e7cfe7f](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/600/commits/e7cfe7f9e8a0735353380428eab54da67e00d7dd)